locator/token_metadata.cc

```diff
@@ -534,16 +535,15 @@ class token_metadata_impl final {
     }
 
 #endif
+    // Returns nodes that are officially part of the ring. It does not include
+    // node that is still joining the cluster.
```

Comment:
Not being an expert in our gossiper lingo makes me wondering what does "still joining the cluster" mean. Are nodes that have tokens are still being streamed to considered "still joining the cluster" or are they "officially part of the ring"? I'm guessing the second is true, but it's ambigous with the wording you used.

Reply:
I see @kbr- had a similar question below, which made me think that my guess wasn't even true - nodes which are still streaming maybe aren't listed. This definitely needs to be explained better in the comment.

Reply:
If a node is still streaming during bootstrap it is not officially part of the ring.

Reply:
That's why I added the second part "It does not include node that is still joining the cluster" to state that node which is joining the cluster is not include in the list.

Reply:
I added more comments in v3. Hope it is more clear now.